### PR TITLE
Changing resource limits

### PR DESCRIPTION
**Pull request by genievot**
Cpu 1 and Memory 2 somehow not working with okteto cloud. So changing it.

### CODE REVIEW

File: tensorflow-notebook/values-okteto.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ jupyter:
   password: okteto
   resources:
     limits:
-      cpu: 1
-      memory: 2
+      cpu: 1.25
```
Comment (pchico83) [inline tensorflow-notebook/values-okteto.yaml:5]: This would not run with the new free tier. The default value should run in the free tier too

Comment (genievot) [reply tensorflow-notebook/values-okteto.yaml:5]: I tried with default one but in okteto cloud with this config it creates error.

Comment (pchico83) [reply tensorflow-notebook/values-okteto.yaml:5]: ouch! there is a typo, it should be `memory: 2Gi`, not `memory: 2`.
Could you update that?

Comment (genievot) [reply tensorflow-notebook/values-okteto.yaml:5]: ok

**Pull request merged**